diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/TrivialPropertyAccessorsWarnTest.kt

```diff
@@ -74,4 +74,18 @@ class TrivialPropertyAccessorsWarnTest : LintTestBase(::TrivialPropertyAccessors
                 """.trimMargin()
         )
     }
+
+    @Test
+    @Tag(TRIVIAL_ACCESSORS_ARE_NOT_RECOMMENDED)
+    fun `should not trigger on getter without braces`() {
```

Comment:
It should or it should not?
```suggestion
    fun `should trigger on getter without braces`() {
```

Reply:
Definetly should